models/tmp/stg_recharge__order_tmp.sql

```diff
@@ -1,3 +1,8 @@
 
 select *
-from {{ var('order') }}
\ No newline at end of file
+from
+{% if var('recharge__using_orders', recharge_source.recharge_does_table_exist('orders')) -%} 
+    {{ var('orders') }}
+{% else -%}
+    {{ var('order') }}
+{%- endif -%}
```

Comment:
Small suggestion to make the code a bit more concise. 
```suggestion
{{ var('orders') if var('recharge__using_orders', recharge_source.recharge_does_table_exist('orders')) else var('order') }}
```

Reply:
Good call. Added!